apps/minifront/src/components/dashboard/assets-table/index.tsx

```diff
@@ -11,13 +11,11 @@ import {
   TableRow,
 } from '@penumbra-zone/ui/components/ui/table';
 import { ValueViewComponent } from '@penumbra-zone/ui/components/ui/tx/view/value';
-import { throwIfPraxNotConnectedTimeout } from '@penumbra-zone/client/prax';
 import { EquivalentValues } from './equivalent-values';
 import { Fragment } from 'react';
 import { shouldDisplay } from './helpers';
 
 export const AssetsLoader: LoaderFunction = async (): Promise<BalancesByAccount[]> => {
-  await throwIfPraxNotConnectedTimeout();
   return await getBalancesByAccount();
```

Comment:
I don't know the exact reason why `throwIfPraxNotConnectedTimeout()` was added to `AssetsLoader`. 

It was this call that caused `PraxNotConnectedError` to be thrown before `PraxNotInstalledError`.

We have no guarantee that the balance load will be earlier than any other prax request, so I see no reason why we shouldn't just remove the `throwIfPraxNotConnectedTimeout()` call

At least I didn't see it break anything

Reply:
there are several of these calls among other loaders. i tried removing all of them and did see failures. i was unsure why some loaders seem to be executing out of order or in unexpected contexts

will fully review this pr soon

Reply:
Ah, missed that there are other loaders that call `throwIfPraxNotConnectedTimeout()`
I'm going to re-watch this PR

Reply:
loaders without `throwIfPraxNotConnectedTimeout` will block page state until their requests succeed or time out. the throws in `LayoutLoader` were prevented from surfacing the correct error page due to the blocking loaders.